CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to teamcity-rest-client library will be documented in this file.
 
+## [1.11.0] - 2020-08-02
```

Comment:
Please use [Unreleased] section for not yet released changes to avoid confusion. And we don't know release date for the next release so it's better to omit it.